src/Other/Helpers/Storage.php

```diff
@@ -0,0 +1,57 @@
+<?php
+
+namespace Rubix\ML\Other\Helpers;
+
+use League\Flysystem\Adapter\Local;
+use League\Flysystem\Filesystem;
+use League\Flysystem\FilesystemInterface;
+use League\Flysystem\Memory\MemoryAdapter;
+
+/**
+ * Storage
+ *
+ * A helper class providing functions relating to storage/persistence.
+ *
+ * @category    Machine Learning
+ * @package     Rubix/ML
+ * @author      Andrew DalPino
+ */
+class Storage
+{
+    /**
+     * Returns a Filesystem instance backed by the local filesystem.
+     *
+     * @param string $root
+     * @param array<mixed> $config
+     * @return FilesystemInterface
+     */
+    public static function local(string $root = '/', array $config = []) : FilesystemInterface
+    {
+        return new Filesystem(new Local($root), $config);
+    }
+
+    /**
+     * Alias of Storage::local
+     * @see Storage::local()
+     *
+     * @param string $root
+     * @param array<mixed> $config
+     * @return FilesystemInterface
+     */
+    public static function filesystem(string $root = '/', array $config = []) : FilesystemInterface
+    {
+        return self::local($root, $config);
+    }
+
+    /**
+     * Returns a Filesystem instance backed by an in-memory mock filesystem.
+     * Suitable for use in tests or when long-term persistence isn't required.
+     *
+     * @param array<mixed> $config
+     * @return FilesystemInterface
+     */
+    public static function memory(array $config = []) : FilesystemInterface
```

Comment:
I'm undecided about this method.. I like that it shows a different `FilesystemInterface` implementation, but it may not provide much real-world use? if we remove this method we could also remove the `league/flysystem-memory` dependency too. 🤷‍♂️